### ISSUE TITLE
Running `rolldown` from the CLI doesn't exit after the build ends.

### ISSUE THREAD

**Issue by hyf0** [rolldown/rolldown#1274]

**Comment by hyf0** [comment]
We have a workaround fix in https://github.com/rolldown/rolldown/pull/1097. The final solution is now bing blocked by https://github.com/napi-rs/napi-rs/pull/2072

**Comment by sxzz** [comment]
I've noticed that it works well when there are no functions in either the input or output options.

```js
bindingInputOptions.onLog = null
normalizedOutputOptions.banner = normalizedOutputOptions.footer = normalizedOutputOptions.sourcemapIgnoreList = undefined
```

**Comment by sxzz** [comment]
This issue can be closed @hyf0 (PR #2135)

**Comment by hyf0** [comment]
Thanks for reminding. Closed by #2135.
